tests/unit/list-utils-test.js

```diff
@@ -663,7 +667,7 @@ describe('Unit: list-utils', function () {
           valueAttribute: 'secret',
           query: {
             booleanFlag: true,
-            universe: '${../universe}'
+            universe: '${./universe}'
```

Comment:
`${./universe}` is correct as the `findValue` utility in `bunsen-core` doesn't behave like the path format followed by `conditions`. Oddly enough, both `../` and `./` will both succeed when the value object is one-level deep.